demo/src/screens/componentScreens/IconScreen.tsx

```diff
@@ -36,6 +38,20 @@ const IconScreen = () => {
       </View>
       <GradientSlider type={GradientSlider.types.HUE} color={color as string} onValueChange={setColor}/>
       <Text marginT-s2>Custom color: {color || '#000000'}</Text>
+
+      <View marginV-s5>
+        <Text marginR-s2 text80BO>
+          Badge Settings:
+        </Text>
+        <View marginT-s3 row>
+          <Text marginR-s2>Use Badge:</Text>
+          <Switch value={useBadge} onValueChange={setUseBadge}/>
```

Comment:
renderBooleanOption function will render both Text and Switch from ExampleScreenPresenter